test/collectionUtils/ExtractDataTypeTransformer.test.ts

```diff
@@ -0,0 +1,27 @@
+import DataType from "../../src/model/dataTypes/DataType";
+import EntityImports from "../../src/model/dataTypes/EntityImports";
+import EntityDefinition from "../../src/model/dataTypes/EntityDefinition";
+import EntityMetadata from "../../src/model/dataTypes/EntityMetadata";
+import MemberVariable from "../../src/model/dataTypes/MemberVariable";
+import ExtractDataTypeTransformer from "../../src/collectionUtils/ExtractDataTypeTransformer";
+
+describe("ExtractDataTypeTransformer", () => {
+  it("should return the DataType for a given EntityDefinition", () => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5966907)